.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
```

Comment:
If this is to be build tested against multiple jvms, can you also include openjdk7?